CONTRIBUTING.md

```diff
@@ -106,6 +106,6 @@ intellectual contributions to the work.
 
 Note that there is no co-authorship expectation for those publishing
 about use of libCEED (versus creation of new features in libCEED), but
-see [`CITING.md`](CITING.md) and use your judgment regarding
-significance of support/advice you may have received in developing
-your use case and interpreting results.
+see the [citing section](https://libceed.readthedocs.io/en/latest/gettingstarted/#how-to-cite)
+and use your judgment regarding significance of support/advice you may have
```

Comment:
I believe we are moving towards one sentence per line (and one line per sentence) for md and rst files to minimize diffs in future commits.

Reply:
Yeah sorry. I have to remember to not maximize my editor's window when editing those files. Otherwise it will show the entire (possibly very very long) line and for me it's a bit hard to read and follow what I'm writing.